pkg/api/acl.go

```diff
@@ -1,10 +1,15 @@
 package api
 
-import "github.com/grafana/grafana/pkg/models"
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/dashboards"
+	"github.com/grafana/grafana/pkg/models"
+)
 
 // updateDashboardACL updates a dashboard's ACL items.
 //
 // Stubbable by tests.
-var updateDashboardACL = func(hs *HTTPServer, dashID int64, items []*models.DashboardAcl) error {
-	return hs.SQLStore.UpdateDashboardACL(dashID, items)
+var updateDashboardACL = func(s dashboards.Store, ctx context.Context, dashID int64, items []*models.DashboardAcl) error {
```

Comment:
Nitpick: Do we want to consistently place context.Context in the first parameter position? (I'd prefer that you'd merge this PR without fixing this if it takes you more than a minute to change this or if you at all disagree)

```suggestion
var updateDashboardACL = func(ctx context.Context, s dashboards.Store, dashID int64, items []*models.DashboardAcl) error {
```

Reply:
Oh, agree - context is ALWAYS first parameter. Should've commented on this before approving :)